lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,134 @@
+# encodeing: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Login
+  def login(options)
+    options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure'])
+
+    return Login::ComplianceServer.login(options) if options['server_type'] == 'compliance'
```

Comment:
Will you be doing the Automate login here too? Probably worth Zooming to review the expected flow, etc. would be good when you're ready :) This mid-method return with no comments as to why is confusing me as to your intentions for this method and how Automate login will be handled.

Reply:
This redirects to the `login` method of the `ComplianceServer` module if the server type is a Compliance server. I was returning here, since this will complete the login. Initially I had it just `exit 0` at the end, but that was causing me issues with the Unit tests.

Will investigate.

Reply:
I'd second @adamleff here - If you're switching between two sets of logic, and you've encapsulated one in a module, I'd expect the other logic - for Automate login - to also be encapsulated in a module (presumably Login::AutomateServer).

Reply:
Originally I thought only to encapsulate the Compliance server stuff because it would be deprecated in the future leaving only `Login`...but the future is not now. I see your point.

Fixed.

Reply:
@adamleff let me know what you think of the flow now that I've modified the name spacing.